crates/api_crud/src/post/create.rs

```diff
@@ -53,6 +54,7 @@ impl PerformCrud for CreatePost {
     let local_user_view =
       get_local_user_view_from_jwt(&data.auth, context.pool(), context.secret()).await?;
     let local_site = LocalSite::read(context.pool()).await?;
+    check_user_approved(&local_user_view, &local_site)?;
```

Comment:
So they can only comment, but not post? I spose it does make it simpler, then you don't have to have a post review queue... but it does seem arbitrary. A troll can spam comments just as much as they could spam posts.